server/app_env/_base.py

```diff
@@ -91,3 +91,6 @@ class Config:
   MIN_STAT_VAR_GEO_COVERAGE = 10
   # NL Bad words file.
   BAD_WORDS_FILE = 'nl_bad_words.txt'
+  # Whether to disable BigQuery from instance. This is primarily used for
+  # accessing the observation browser pages.
+  DISABLE_BQ = False
```

Comment:
Is it possible to make it disabled by default and enable it only for main dc?

Also, if we go with that, consider naming it `ENABLE_BQ`.

Reply:
Good idea, thanks! I switched to ENABLE_BQ and only added for main dc + test environments